src/logging.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import * as winstonLogger from './Adapters/Logger/WinstonLogger';
+
+let logger = winstonLogger.logger;
+export function setLogger(aLogger) {
+  if (!aLogger) {
+    console.log('NO LOGGER SET!');
+  }
+  logger = aLogger;
+}
+
+function getLogger() {
+  return logger;
+}
+
+export function addTransport(transport) {
```

Comment:
Is this still useful? I think if we set a different logger then it won't do what's expected. Could we instead move inside a logger adapter implementation? Also, transport seems like a winston specific feature